src/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from dll import DoublyLinkedList as DLL
+
+
+@pytest.fixture()
```

Comment:
Great job using fixtures! Keep expanding your use and you will find it will save you a lot of repitition. Consider making an empty list fixture, a fixture with one item, a fixture with lots of items, etc. Then if you want to take it to the next level, instead of just returning the instance from your fixture, you can return a dictionary of data, in which the instance is just one of many values, along with the expected pop value, expected pop error, expected length after pop, etc etc. All this stuff can be defined at the fixture level.